### DISCUSSION TITLE
udm-iptv doesn't start: status=255/EXCEPTION

### DISCUSSION THREAD

**Discussion by HamburgerPlsdotexe** [fabianishere/udm-iptv#217] [Q&A] [answered]
Hey there everyone! 

I have an ITV-subscription on KPN and tried to get the script to work with no luck.

Whenever I try to start udm-iptv, systemctl status will show that the program exited with an exception (status=255). 
Further investigation shows that igmpproxy exits with MRT_INIT failed; Errno(92): Protocol not available.

I'm not entirely sure what I'm doing wrong as I've enabled IGMP snooping on the VLAN (4) which the IPTV-decoder is located in as well as the LAN.

My settings within the udm conf are the following:

`IPTV_WAN_INTERFACE="eth8"

IPTV_WAN_VLAN="4"

IPTV_WAN_VLAN_INTERFACE="iptv"

IPTV_WAN_RANGES="213.75.0.0/16  217.166.0.0/16"

IPTV_WAN_DHCP_OPTIONS="-O staticroutes -V IPTV_RG"

IPTV_LAN_INTERFACES="br4" // br4 should be ethernet port 4, right? The port which my decoder is plugged in to?

IPTV_IGMPPROXY_DISABLE_QUICKLEAVE="false"

IPTV_IGMPPROXY_DEBUG="false"
`

Does anyone have an idea of what might be the issue? I will provide more information if necessary!

Thank you for your time!

**Comment by fabianishere** [comment, marked as answer]
What firmware version do you have installed? The error indicates multicast routing is not supported by the kernel.

Also, `br4` refers to your internal VLAN 4.

**Reply by HamburgerPlsdotexe** [reply]
Updated to 3.0. It is somewhat working now, the stream is getting constantly interrupted after a few seconds and then a stb-nmc-400 shows up :/

Zapping to other channels yields the same results
